go/libraries/doltcore/env/actions/clone.go

```diff
@@ -32,9 +32,9 @@ import (
 	"github.com/dolthub/dolt/go/libraries/utils/filesys"
 	"github.com/dolthub/dolt/go/libraries/utils/iohelp"
 	"github.com/dolthub/dolt/go/libraries/utils/strhelp"
+	stchunks "github.com/dolthub/dolt/go/store/chunks"
```

Comment:
why the package alias here?

Reply:
There's a `chunks` variable on line 94 that conflicts. Can change the name of that if that's preferred?